app/src/main/java/io/intrepid/dictionary/DictionaryActivity.kt

```diff
@@ -0,0 +1,25 @@
+package io.intrepid.dictionary
+
+import android.os.Bundle
+import android.support.v7.app.AppCompatActivity
+import kotlinx.android.synthetic.main.activity_main.*
+import kotlinx.android.synthetic.main.content_main.*
+
+class DictionaryActivity : AppCompatActivity(), DictionaryView {
+
```

Comment:
I don't include a newline at the top here, but let me know if it has some utility/role in Kotlin